backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/s3/BucketService.java

```diff
@@ -39,15 +50,22 @@
  * <p>Usage:</p>
  * This service is used by invoking either {@code publishPublicNorm} or {@code publishPrivateNorm}
  * to upload a norm document to the respective storage. In case of a failure during the upload,
- * a {@link BucketPublishException} is thrown, encapsulating the bucket name and norm details.
+ * a {@link BucketException} is thrown, encapsulating the bucket name and norm details.
  *
  * @see PublishPublicNormPort
  * @see PublishPrivateNormPort
- * @see BucketPublishException
+ * @see BucketException
  */
 @Service
 @Slf4j
-public class BucketService implements PublishPublicNormPort, PublishPrivateNormPort {
+public class BucketService
+  implements
+    PublishPublicNormPort, PublishPrivateNormPort, DeletePublicNormPort, DeletePrivateNormPort {
+
+  private static final String PUT = "PUT";
+  private static final String DELETE = "DELETE";
+  private static final String CHANGELOG_KEY = "changelog.json";
```

Comment:
are we using the same changelog file all the time? The agreement with the portal was that we write a new changelog file whenever we change some files. So eg. `2023-12-01.json` on one day and then `2023-12-02.json` at the next day and so on.